src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -384,7 +384,7 @@ private async Task<TestLspServer> CreateTestLspServerAsync(EditorTestWorkspace w
             InitializationOptions? options, string? workspaceKind, bool mutatingLspWorkspace, TestComposition? composition = null)
         {
             var workspace = new EditorTestWorkspace(
-                composition ?? Composition, workspaceKind, configurationOptions: new WorkspaceConfigurationOptions(EnableOpeningSourceGeneratedFiles: true), supportsLspMutation: mutatingLspWorkspace);
+                composition ?? Composition, workspaceKind, configurationOptions: new WorkspaceConfigurationOptions(ValidateCompilationTrackerStates: true), supportsLspMutation: mutatingLspWorkspace);
```

Comment:
why was ValidateCompilationTrackerStates: true added?

Reply:
it should have been there from teh beginning.  it's helpful for tracking down bugs that jason cares about.